### ISSUE TITLE
Has the syntax for Shortcuts changed?  This was working before but not anymore...

### ISSUE THREAD

**Issue by hchung-PA** [AlexxIT/WebRTC#239]
```
        card:
          type: custom:webrtc-camera
          entity: camera.front_door
          mse: false
          title: Front Door
          ui: true
          shortcuts:
            services:
              - name: Stop
                icon: mdi:stop
                service: eufy_security.stop_p2p_livestream
                service_data:
                  entity_id: camera.front_door
```

Had this code working with a stop button to stop the Eufy stream but this button doesn't show up anymore for whatever reason....

**Comment by TomHart** [comment]
Seeing similar, trying to set them up for the first time, and the example on the README doesn't display anything, and can't see anything in the DOM either

**Comment by supiiik** [comment]
I have same problem :(

**Comment by Zeunas** [comment]
I believe there is already a fix for the upcoming 2.2.1 as commented on this issue #212. Is there a timescale for the release? Quite an handy option to have.

For now it only seems to work without the below but then buttons become overlapped with the Title:

> top: 0
> left: 0
> orientation: vertical
> services:

**Comment by AlexxIT** [comment]
All fixes in latest release https://github.com/AlexxIT/WebRTC/releases/tag/v2.3.0
Please reopen issue if it still actual
